template/config.yaml

```diff
@@ -4,6 +4,8 @@ settings:
   docker:
 {%- if accelerator == 'gpu' %}
     parent_image: 'huggingface/transformers-pytorch-gpu'
+    build_options:
```

Comment:
is this needed?

Reply:
This doesnt work either it has to be 

```yaml
    build_options: 
      platform: 'linux/amd64'
```

Reply:
Unfortunately yes for m1 users, because it tries to pull an arm image which doesn't exist

Reply:
yes but still the actual command is wrong i think (see second comment)